tests/Unit/Media/ClipTest.php

```diff
@@ -0,0 +1,68 @@
+<?php
+namespace Tests\FFMpeg\Unit\Media;
+
+use FFMpeg\Media\Clip;
+
+class ClipTest extends AbstractMediaTestCase
+{
+
+    /**
+     * @dataProvider provideBuildOptions
+     */
+    public function testBuildCommand($startValue, $durationValue, $commands)
+    {
+        $configuration = $this->getMock('Alchemy\BinaryDriver\ConfigurationInterface');
```

Comment:
Please use `\Alchemy\BinaryDriver\ConfigurationInterface::class` (and so on) instead. :)

Reply:
sorry I do not understand :(

Reply:
Unrelated in this code branch. There is nothing to improve in this codestyle :)